### PR TITLE
Convenient plugins that handles cats typeclasses.

### PR DESCRIPTION
**Pull request by VladPodilnyk**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -573,6 +573,7 @@ lazy val `fundamentals-bio` = project.in(file("fundamentals/fundamentals-bio"))
       "org.typelevel" %% "cats-core" % V.cats % Optional,
       "org.typelevel" %% "cats-effect" % V.cats_effect % Optional,
       "dev.zio" %% "zio" % V.zio % Optional,
+      "dev.zio" %% "zio-interop-cats" % V.zio_interop_cats % Optional,
```
Comment (neko-kai) [inline build.sbt]: Could you add this just to `distage-framework`, not everywhere?

**Review by neko-kai — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (VladPodilnyk) [reply build.sbt]: Yup, a few sec, please)

Comment (VladPodilnyk) [reply build.sbt]: also, fix warnings while creating timer
```
 make[Timer[Task]].from(Timer[Task])
```

File: build.sbt [hunk 2]
```diff
@@ -1507,7 +1507,8 @@ lazy val `distage-framework` = project.in(file("distage/distage-framework"))
       "org.typelevel" %% "cats-effect" % V.cats_effect % Optional,
       "dev.zio" %% "zio" % V.zio % Optional,
       "io.monix" %% "monix-bio" % V.monix_bio % Optional,
-      "org.scala-lang" % "scala-reflect" % scalaVersion.value % Provided
+      "org.scala-lang" % "scala-reflect" % scalaVersion.value,
```
Comment (neko-kai) [inline build.sbt]: @VladPodilnyk Could please you return `Provided` here and use Optional in interop-cats, not Provided?

Comment (VladPodilnyk) [reply build.sbt]: Oh, yeah, I've messed this up a lil bit.
Fixed.

**Comment by neko-kai** [comment]
@VladPodilnyk Thank you! 🙏

**Comment by VladPodilnyk** [comment]
Not at all 🙂

**Pull request merged**